src/ray/raylet/placement_group_resource_manager_test.cc

```diff
@@ -205,7 +215,7 @@ TEST_F(NewPlacementGroupResourceManagerTest, TestNewMultipleBundlesCommitAndRetu
   remaining_resource_instance = remaining_resource_scheduler->GetLocalNodeResources();
   ASSERT_TRUE(update_called_);
   ASSERT_TRUE(delete_called_);
-  CheckRemainingResourceCorrect(remaining_resource_instance);
+  // CheckRemainingResourceCorrect(remaining_resource_instance);
```

Comment:
Can you uncomment this and change the function to take into account of remaining bundle resource information?

Reply:
Goot catch! That's commented by accidiental.